Commander X16 Programmer's Reference Guide.md

```diff
@@ -16,127 +16,137 @@ This describes the "Proto2" board revision and the emulator/ROM versions r39 and
-            * [Function Name: FB_filter_pixels](#function-name-fb_filter_pixels)
-            * [Function Name: FB_move_pixels](#function-name-fb_move_pixels)
-         * [Graphics](#graphics)
-            * [Function Name: GRAPH_init](#function-name-graph_init)
-            * [Function Name: GRAPH_clear](#function-name-graph_clear)
-            * [Function Name: GRAPH_set_window](#function-name-graph_set_window)
-            * [Function Name: GRAPH_set_colors](#function-name-graph_set_colors)
-            * [Function Name: GRAPH_draw_line](#function-name-graph_draw_line)
-            * [Function Name: GRAPH_draw_rect](#function-name-graph_draw_rect)
-            * [Function Name: GRAPH_move_rect](#function-name-graph_move_rect)
-            * [Function Name: GRAPH_draw_oval](#function-name-graph_draw_oval)
-            * [Function Name: GRAPH_draw_image](#function-name-graph_draw_image)
-            * [Function Name: GRAPH_set_font](#function-name-graph_set_font)
-            * [Function Name: GRAPH_get_char_size](#function-name-graph_get_char_size)
-            * [Function Name: GRAPH_put_char](#function-name-graph_put_char)
-         * [Console](#console)
-            * [Function Name: console_init](#function-name-console_init)
-            * [Function Name: console_put_char](#function-name-console_put_char)
-            * [Function Name: console_put_image](#function-name-console_put_image)
-            * [Function Name: console_get_char](#function-name-console_get_char)
-            * [Function Name: console_set_paging_message](#function-name-console_set_paging_message)
-         * [Other](#other)
-            * [Function Name: memory_fill](#function-name-memory_fill)
-            * [Function Name: memory_copy](#function-name-memory_copy)
-            * [Function Name: memory_crc](#function-name-memory_crc)
-            * [Function Name: memory_decompress](#function-name-memory_decompress)
-            * [Function Name: entropy_get](#function-name-entropy_get)
-            * [Function Name: monitor](#function-name-monitor)
-            * [Function Name: screen_mode](#function-name-screen_mode)
-            * [Function Name: screen_set_charset](#function-name-screen_set_charset)
-            * [Function Name: JSRFAR](#function-name-jsrfar)
-   * [Math Library](#math-library)
-   * [Machine Language Monitor](#machine-language-monitor)
-   * [Memory Map](#memory-map)
-      * [Banked Memory](#banked-memory)
-      * [ROM Allocations](#rom-allocations)
-      * [RAM Contents](#ram-contents)
-         * [Zero Page](#zero-page)
-         * [Banking](#banking)
-      * [I/O Area](#io-area)
-   * [Video Programming](#video-programming)
-   * [Sound Programming](#sound-programming)
-   * [I/O Programming](#io-programming)
-   * [Real-Time-Clock Programming](#real-time-clock-programming)
+- [Commander X16 Programmer's Reference Guide](#commander-x16-programmers-reference-guide)
```

Comment:
This can't change in formatting, since it's auto-generated by the tool referenced in line 17